### PR TITLE
add Graphul

### PR DESCRIPTION
**Pull request by SamuelBonilla**
An Express-inspired web framework written in Rust. https://graphul-rs.github.io/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -433,10 +433,10 @@ See also [A comparison of operating systems written in Rust](https://github.com/
...
 * [LemmyNet/lemmy](https://github.com/LemmyNet/lemmy) — A link aggregator / reddit clone for the fediverse [![Build Status](https://cloud.drone.io/api/badges/LemmyNet/lemmy/status.svg)](https://cloud.drone.io/LemmyNet/lemmy)
-* [libreddit](https://github.com/libreddit/libreddit) - An alternative private front-end to Reddit
 * [MASQ-Project/Node](https://github.com/MASQ-Project/Node) — MASQ Node software provides a decentralized mesh-network of nodes for global users to access normal internet content - next evolution of tech beyond Tor & VPN [![build badge](https://github.com/MASQ-Project/Node/actions/workflows/ci-matrix.yml/badge.svg)](https://github.com/MASQ-Project/Node/actions)
 * [Plume-org/Plume](https://github.com/Plume-org/Plume) — ActivityPub federating blogging application [![build badge](https://api.travis-ci.org/Plume-org/Plume.svg?branch=master)](https://travis-ci.org/Plume-org/Plume)
 * [Revolt/backend](https://github.com/revoltchat/backend) - User-first chat platform built with modern web technologies.
+* [spikecodes/libreddit](https://github.com/spikecodes/libreddit) - An alternative private front-end to Reddit
```
Comment (palfrey) [inline README.md:439]: This reverts the fix from master. Please change it back!

**Comment by palfrey** [comment]
Also, as the link isn't to a github repo, but the repo has enough stars, please add an entry for the link to the bottom of https://github.com/rust-unofficial/awesome-rust/blob/main/src/main.rs#L68 noting this

File: README.md [hunk 2]
```diff
@@ -1525,6 +1527,7 @@ See also [Are we web yet?](https://www.arewewebyet.org) and [Rust web framework
   * [hyperium/hyper](https://github.com/hyperium/hyper) — an HTTP implementation [![CI](https://github.com/hyperium/hyper/workflows/CI/badge.svg?branch=master)](https://github.com/hyperium/hyper/actions?query=workflow%3ACI)
   * [seanmonstar/reqwest](https://github.com/seanmonstar/reqwest) — an ergonomic HTTP Client for Rust. [![build badge](https://api.travis-ci.org/seanmonstar/reqwest.svg?branch=master)](https://travis-ci.org/seanmonstar/reqwest)
 * HTTP Server
+  * [Graphul](https://graphul-rs.github.io/) — An Express-inspired web framework written in Rust. [![build badge]
```
Comment (palfrey) [inline README.md:1530]: `[![build badge]` isn't a valid badge!

Comment (SamuelBonilla) [reply README.md:1530]: Ok

File: README.md (hunk 1 above)
Comment (Israel-Laguan) [reply README.md:439]: Maybe you need to rebase, seems like your base was old

File: README.md [hunk 3]
```diff
@@ -559,6 +559,8 @@ See also [Foreign Function Interface](https://doc.rust-lang.org/book/first-editi
   * [rust-lang/rust-bindgen](https://github.com/rust-lang/rust-bindgen) — A Rust bindings generator
 * Erlang
   * [rusterlium/rustler](https://github.com/rusterlium/rustler) — safe Rust bridge for creating Erlang NIF functions [![build badge](https://api.travis-ci.org/rusterlium/rustler.svg?branch=master)](https://travis-ci.org/rusterlium/rustler)
+* Haskell
```
Comment (Israel-Laguan) [inline README.md:562]: This change should not be part of this PR. If you needed to add this, create a new PR
